### PR TITLE
Added info about and link to Unity GDK

### PR DESCRIPTION
**Pull request by ElleEss**
As requested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-- *GitHub repository*: [https://github.com/spatialos/StarterProject](https://github.com/spatialos/StarterProject)
+- *GitHub repository*: [github.com/spatialos/StarterProject](https://github.com/spatialos/StarterProject)
+
 
----
 
 ## Introduction
 
-This is a SpatialOS starter project with useful core features that you can extend to build your own SpatialOS application.
+This is a [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos) starter project to use with the [Unity SDK](https://github.com/spatialos/UnityGDK) with useful core features that you can extend to build your own SpatialOS application.
```
Comment (anne-edwards) [inline README.md]: I think the second link here should go to /spatialoos/UnitySDK, rather than UnityGDK?

Comment (ElleEss) [reply README.md]: Yes! Got GDK on brain! Thanks.

Comment (ElleEss) [reply README.md]: Done.

**Review by anne-edwards — APPROVED** [review]
Looks good!

**Pull request merged**